sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -572,7 +572,6 @@
         // Set required defaults
         setTopicPartitions(Collections.emptyList());
         setConsumerFactoryFn(KafkaIOUtils.KAFKA_CONSUMER_FACTORY_FN);
```

Comment:
We also have a redundant `setConsumerFactoryFn(KafkaIOUtils.KAFKA_CONSUMER_FACTORY_FN);`